containers/run.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# launch model server
+/model/src/model_server/run_model_server.sh &
+
+# ASCII art
+printf "Running ALLAN bot\n"
+printf "    ___    __    __    ___    _   __\n"
+printf "   /   |  / /   / /   /   |  / | / /\n"
+printf "  / /| | / /   / /   / /| | /  |/ / \n"
+printf " / ___ |/ /___/ /___/ ___ |/ /|  /  \n"
+printf "/_/  |_/_____/_____/_/  |_/_/ |_/   \n"
+printf "\n"
+
+export PYTHONPATH=$PYTHONPATH:/model/src/model_server/research/
+
+# create random game id
+random_id=$(shuf -i 1-10000000 -n 1)
+game_id="ALLAN_game_${random_id}"
```

Comment:
let's have some prefix as well to game id. And could you move the test container related files in a different folder like test_env or something so that the purpose of having another run.sh, etc stays clear?

Reply:
What kinds of prefix are you thinking?